src/components/Header.js

```diff
@@ -4,28 +4,28 @@ import { Link } from 'react-router-dom';
 const Header = (props) => {
   return (
     <header>
-      <div>
+      <div className='header-text'>
         <Link to='/'>
```

Comment:
Can the `<Link />` component take a `className` attribute to minimize the # of `<div />`'s we need?

Reply:
Yup, from looking at the source code for Link, it will pass any props given to it onto the rendered anchor tag.